sierra_adapter/sierra_bib_merger/src/test/scala/uk/ac/wellcome/platform/sierra_bib_merger/SierraBibMergerFeatureTest.scala

```diff
@@ -279,8 +291,11 @@ class SierraBibMergerFeatureTest
     withLocalSqsQueue { queue =>
       withLocalS3Bucket { bucket =>
         withLocalDynamoDbTable { table =>
-          val flags = sqsLocalFlags(queue) ++ s3LocalFlags(bucket) ++ dynamoDbLocalEndpointFlags(
-            table)
+          val flags = sqsLocalFlags(queue) ++ s3LocalFlags(
+            bucket,
+            namespace = "vhs") ++ dynamoDbLocalEndpointFlags(
+            table,
+            namespace = "vhs")
```

Comment:
Better than namespacing would be to take a leaf out of #1944, and have a vhsLocalFlags instead.